### ISSUE TITLE
Most likely broken in Nether and End / other dimensions

### ISSUE THREAD

**Issue by BlayTheNinth** [blay09/ForgivingVoid#29]
### Minecraft Version

1.18.x

### Mod Loader

Forge

### Mod Loader Version

dev

### Mod Version

dev

### Describe the Issue

Overworld minimum level changed, but probably End and Nether stayed the same? Might be a per-dimension setting now that I need to use, instead of having only a single config option for all

**Comment by mcenderdragon** [comment]
the level now has a getMinHeight and its per dimension and can dynam,icly chasnge via datapacks, also a bug in 1.17

**Comment by Joeppie** [comment]
Hello, the issue still seems to occur when running forgivingvoid-forge-1.18.1-6.0.1.jar as downloaded from curseforge (https://www.curseforge.com/minecraft/mc-mods/forgiving-void/files/3606213) for 1.18.2. This is the version that was created the very same day this issue was closed. Was the fix missing, or was it never fixed for this version?

Somewhat offtopic, but can we expect a fixed version? Right now anyone downloading that version is going to have a **bad** time, getting disconnected and getting stuck in a disconnect loop to servers. Removing forgiving void from only client or server is not going to solve the problem, so makes it kind of painful for anyone who happens to start running the mod and after testing on overworld believes it to be working (as I did.)
